entries/o/opencollective.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Open Collective": {
+    "domain": "opencollective.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://docs.opencollective.com/help/fiscal-hosts/payouts/two-factor-authentication-for-payouts#enabling-2fa-for-login",
+    "keywords": [
+      "other"
```

Comment:
I suggest not using the keyword `other` for new entries whenever possible. How about `payments` or `finance`, or maybe both?